Makefile

```diff
@@ -53,5 +53,4 @@ install: primus-os.bin
 	sudo cp $< /boot/primus-os.bin
 
 clean:
-	rm -f *.o primus-os primus-os.iso primus-os.bin $(OBJ_DIR)/*.o
-	rm -rf iso
\ No newline at end of file
+	rm -f *.o primus-os primus-os.iso primus-os.bin $(OBJ_DIR)/*.o iso
```

Comment:
you must add the r flag (`rm -rf`) to make it recursive since it is deleting a directory.